applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -2065,6 +2065,7 @@ else if ( "preloadWidgetAdView".equals( call.method ) )
         {
             String adUnitId = call.argument( "ad_unit_id" );
             String adFormatStr = call.argument( "ad_format" );
+            boolean isAdaptiveBannerEnabled = call.argument( "isAdaptiveBannerEnabled" );
```

Comment:
Why are some snake_case and some camel case?

Reply:
My bad.   Fixed and tested.